circle.yml

```diff
@@ -24,7 +24,7 @@ general:
 
 deployment:
   release:
-    tag: /(([a-z]+)-)?([0-9]+)\.([0-9]+)\.([0-9]+)/
+    tag: /(([a-z]+)-)*([0-9]+)\.([0-9]+)\.([0-9]+)/
```

Comment:
- Some expected tags are in my [2nd comment][1]. The reason we allow it to be empty is because umbrella tags are expected to be `0.x.y` (discussed in #2810)
- Where did you imagine the documenting happens? Here or in another file?

[1]: https://github.com/GoogleCloudPlatform/google-cloud-python/pull/2853#issuecomment-266242369